fusor-ember-cli/app/controllers/undercloud-deploy.js

```diff
@@ -132,7 +127,7 @@ export default Ember.Controller.extend(DeploymentControllerMixin, {
 	      success: function(response) {
 	        console.log('api check success');
 	          console.log(response);
-		  if (response['deployed']) {
+		  if (response['deployed'] || response['failed']) {
 		      console.log('deployment finished');
 		      if (response['failed']) {
 			  console.log('deployment failed');
```

Comment:
The indentation in this file makes it really hard to read. Can we clean it up a bit?